edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Http/middleware/AuthenticationMiddleware.cs

```diff
@@ -84,7 +84,7 @@ internal async Task<(bool, string)> AuthenticateRequest(HttpContext context)
             if (clientCertificate != null)
             {
                 IList<X509Certificate2> certChain = context.GetClientCertificateChain();
-                clientCredentials = this.identityFactory.GetWithX509Cert(deviceId, moduleId, string.Empty, clientCertificate, certChain);
+                clientCredentials = this.identityFactory.GetWithX509Cert(deviceId, moduleId, string.Empty, clientCertificate, certChain, Option.None<string>());
```

Comment:
>clientCredentials = this.identityFactory.GetWithX509Cert(deviceId, moduleId, string.Empty, clientCertificate, certChain, Option.None<string>()); [](start = 16, length = 144)

FYI I deleted this file in a pending PR to refactor the HTTP auth, one of us will need to deal with the merge conflict when this payload hits iiot branch.